dask_sql/physical/rel/custom/export_model.py

```diff
@@ -0,0 +1,99 @@
+import pickle
+
+import joblib
+import sklearn
+
+from dask_sql.datacontainer import DataContainer
+from dask_sql.physical.rel.base import BaseRelPlugin
+from dask_sql.utils import convert_sql_kwargs
+
+
+class SqlExportModel(BaseRelPlugin):
+    """
+    Export a trained model into a file using supported model serilization or model serDe.
+
+    Sql:
+        EXPORT MODEL <model_name> WTIH (
+            model_serde = "pickle",
+            location = "model.pkl"
+        )
+    1. Most of the framework support pickle as a serilization
+        example:
+            sklearn
+            Dask ML
+            Pytorch
+    2. To Export a Universal (Framework agnostic model) use mlflow format
+        - A mlflow is a framework which supports different flavours of model serilization, implemented
+        for different ML libraries.
+        - Mlflow model is a self contained directory full of files which helps to load the
+        saved model without any environment Error
+        - To reproduce the environment conda.yaml files are produced while saving the model
+        Files saved as part of mlflow model
+            - conda.yaml (packages required to import saved model)
+            - MLmodel (A artifact about model which contains following)
+                - mlflow function which can load this saved model
+                - model was saved timestmap  etc..
+            - model : a ml model itself
+                - sklearn - model.pkl
+                - xgboost - model.xgb
+                - lightgbm - model.lgb
+    3. Export into ONNX (open standard ) -
+         - Needs input column names and their type to serilizes the model
```

Comment:
It's good to have this as a reminder, that this might come in the future but I would not include this in the documentation (otherwise it looks as if we can do it already). If you want, you can include a `TODO`.

Reply:
Added as a TODO